types/ckeditor__ckeditor5-utils/src/fastdiff.d.ts

```diff
@@ -0,0 +1,85 @@
+import { DeleteChange, InsertChange } from "@ckeditor/ckeditor5-utils/src/difftochanges";
```

Comment:
```suggestion
import { DeleteChange, InsertChange } from "./difftochanges";
```
shouldn't this one be relative as well?